packages/react/src/components/Modal/Modal.js

```diff
@@ -416,7 +416,8 @@ export default class Modal extends Component {
         onClick={onRequestClose}
         title={iconDescription}
         aria-label={iconDescription}
-        ref={this.button}>
+        ref={this.button}
+      >
```

Comment:
These changes throughout this file seemed to be added via `husky` during the commit process-- hopefully they fit with this repo's conventions.